app/templates/reminders/reminder/index.hbs

```diff
@@ -0,0 +1,13 @@
+<h2 class="reminder-title">
+  {{model.title}}
+</h2>
+<h4>
+  <p>{{model.body}}</p>
+  <p>{{model.date}}</p>
+
+  {{#link-to 'reminders.reminder.edit'}}
+  <button class='edit-reminder-button'>Edit this reminder</button>
```

Comment:
Indent. Also I think we could probably not wrap everything in a <h4> as much as create a div with class and change the font size using CSS.